codespell_lib/_codespell.py

```diff
@@ -1168,6 +1168,16 @@ def main(*args: str) -> int:
         if is_hidden(filename, options.check_hidden):
             continue
 
+        # skip argument if any piece of the path is in the skip list
```

Comment:
Perhaps I can make this a function to avoid increasing pylint acceptable metrics too much.

Reply:
sure, some `if _match_path_parts(...): continue` or so would help `main` be more readable

Reply:
Done with the suggested name.